node/runtime/src/state_viewer.rs

```diff
@@ -19,9 +18,7 @@ pub struct ViewStateResult {
     pub values: HashMap<Vec<u8>, Vec<u8>>
 }
 
-pub struct StateDbViewer {
-    state_db: Arc<StateDb>,
-}
+pub struct StateDbViewer {}
```

Comment:
this should just be a module if we aren't going to keep any struct members

Reply:
Yeah. Will change that later

Reply:
let's make an issue then

Reply:
Actually what do you think of practice  having structs with no fields just to group methods?

Reply:
I am ambivalent about it. Not sure which one is more idiomatic

Reply:
pretty sure that's the point of a file or module

Reply:
Empty structs are a normal thing in Rust, one can use them in combination with traits. Suppose we had two different ways to implement `StateDbViewer` functionality (e.g. DebugNet and TestNet), then we could create a common trait and make RPC depend on that trait. On the other hand, if had two versions of `StateDbViewer` implemented as two different modules then we would not be able to have RPC that can easily switch between them.

In the current situation, however, we do not have alternative implementations of `StateDbViewer` so it probably does not matter whether it is a module or an empty struct.